src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx

```diff
@@ -0,0 +1,170 @@
+                text: '_t_Deriv Prime is the institutional arm of Deriv Group, and staying true to its guiding principles, effectively democratises liquidity provision to financial institutions._t_',
+            },
+            {
+                id: 2,
+                type: 'text',
+                text: '_t_No matter if you are large, small, or have just started, Deriv Prime will provide top-of-book liquidity solutions with very fast execution at unmatched fair pricing and conditions._t_',
+            },
+        ],
+    },
+    {
+        id: 2,
+        question: '_t_How can I open an Up/Down contract?_t_',
+        answers: [
+            {
+                id: 1,
+                type: 'text',
+                text: '_t_Deriv Prime partners with regulated financial institutions, including:_t_',
+            },
+            {
+                id: 2,
+                type: 'list',
+                list_items: [
+                    '_t_- Brokerage firms_t_',
+                    '_t_- Asset managers_t_',
+                    '_t_- Investment funds_t_',
+                    '_t_- Professional trading groups / professional traders_t_',
+                    '_t_- Corporate clients_t_',
+                    '_t_- Other financial institutions_t_',
+                ],
+            },
+        ],
+    },
+    {
+        id: 3,
+        question: '_t_What is the liquidity solution offering?_t_',
+        answers: [
+            {
+                id: 1,
+                type: 'text',
+                text: '_t_As a liquidity solutions provider, Deriv Prime offers institutional trading services and access to global financial markets through a single point of integration. This encompasses trading assets in the forex, commodities, equities, indices, ETFs, and cryptocurrencies markets._t_',
+            },
+        ],
+    },
+    {
+        id: 4,
+        question: '_t_What is the onboarding procedure?_t_',
+        answers: [
+            {
+                id: 1,
+                type: 'text',
+                text: '_t_To onboard with Deriv Prime’s liquidity solution and trading services, you will need to follow a streamlined process:_t_',
+            },
+            {
+                id: 2,
+                type: 'list',
+                list_items: [
+                    '_t_1. Account application: Contact us to connect with a dedicated support team to guide you through the entire onboarding journey._t_',
+                    '_t_2. Verification: Satisfy all compliance and regulatory requirements including Know Your Business (KYB) requirements._t_',
+                    '_t_3. Margin account settlement: Fund your margin account to meet the minimum balance based on our scalable commercial plan._t_',
+                    '_t_4. Connectivity: Integrate with our platform using your preferred connectivity method._t_',
+                ],
+            },
+        ],
+    },
+]
+
+export const tocFAQDAta: DerivPrimeFAQDataItem[] = [
```

Comment:
can you change this variable name to `snake_case` something like terms_cooperation_data